formspree/users/views.py

```diff
@@ -314,13 +323,8 @@ def stripe_webhook():
         elif event['type'] == 'invoice.payment_failed': # User payment failed
             customer_id = event['data']['object']['customer']
             customer = stripe.Customer.retrieve(customer_id)
-            g.log.info('User payment failed', account=customer.email)
-            send_email(to=customer.email,
-                       subject='[ACTION REQUIRED] Failed Payment for {} {}'.format(settings.SERVICE_NAME,
-                                                                           settings.UPGRADED_PLAN_NAME),
-                       text=render_template('email/payment-failed.txt'),
-                       html=render_template('email/payment-failed.html'),
-                       sender=settings.DEFAULT_SENDER)
+            send_failed_email.delay(4)
```

Comment:
Shouldn't it be `send_failed_email.delay(customer)` here, as per http://docs.celeryproject.org/en/latest/userguide/calling.html#basics?

Reply:
Yes, you're right, this was hard coding for testing I didn't remove